lib/generators/doorkeeper/templates/migration.rb.erb

```diff
@@ -10,9 +10,9 @@ class CreateDoorkeeperTables < ActiveRecord::Migration<%= migration_version %>
       t.timestamps             null: false
     end
 
-    add_index :oauth_applications, :uid, unique: true
+    add_index :oauth_applications, :uid, unique: true, <%= primary_id_option %>
```

Comment:
Why do we need to add this option for `add_index` ?

Reply:
it is not needed, will fixed.